src/leiningen/new/re_frame/shadow-cljs.edn

```diff
@@ -5,25 +5,16 @@
  :builds {:app {:target          :browser
                 :output-dir      "resources/public/js/compiled"
                 :asset-path      "/js/compiled"
-                :dev             {:modules          {:app {:entries [devtools.preload{{#10x?}}
-                                                                     day8.re-frame-10x.preload{{/10x?}}{{#re-frisk?}}
-                                                                     re-frisk.preload{{/re-frisk?}}
-                                                                     {{name}}.core]}}{{#10x?}}
-                                  :compiler-options {:closure-defines {re-frame.trace.trace-enabled?        true
+                :modules         {:app {:init-fn {{name}}.core/init
+                                        :preloads [devtools.preload{{#10x?}}
+                                                   day8.re-frame-10x.preload{{/10x?}}{{#re-frisk?}}
+                                                   re-frisk.preload{{/re-frisk?}}]}}{{#10x?}}
```

Comment:
`:preloads` are `:dev` only. no need for separate `:modules` config for `:dev` and `:release`.